integration/kubernetes/helm-chart/alluxio/templates/_helpers.tpl

```diff
@@ -238,3 +238,35 @@ resources:
             defaultMode: 256
   {{- end }}
 {{- end -}}
+
+{{- define "alluxio.master.readinessProbe" -}}
+readinessProbe:
+  exec:
+    command: ["bin/alluxio-monitor.sh", "master"]
+{{- end -}}
+
+{{- define "alluxio.worker.readinessProbe" -}}
+readinessProbe:
+  exec:
+    command: ["bin/alluxio-monitor.sh", "worker"]
+{{- end -}}
+
+{{- define "alluxio.master.livenessProbe" -}}
```

Comment:
You are right on readiness.

For liveness I think it's "is the process/service up, but not necessarily ready to serve clients" plus "if it's not alive it can/should possibly be restarted, trying to recover". If a liveness probe fails (for a specific number of times), K8s will restart it. That's why I think `fsadmin report` is not a good liveness probe, because oftentimes it's not a recoverable state.